src/main/java/com/ride2go/ridetogtfsconverter/validation/TripValidator.java

```diff
@@ -104,6 +107,32 @@ public void validTrips(List<EntityTrip> trips) {
 		}
 	}
 
+	public void withDurationGreaterThanZero(List<Offer> offers) {
```

Comment:
Why not return the List of offers instead of void? I only ask out of curiosity.

Reply:
Yes, the disadvantage is that you can't for example use `withDurationGreaterThanZero(offers)` as a parameter for another method. Other than that I think it's less work in this method when you don't have to copy the list in the first place. So if a parameter like `offers` is not primitive and is not set to `final`, i think it's fine to use it as a return value.